README.md

```diff
@@ -51,6 +51,21 @@ and also for [Sherpa in CIAO](http://cxc.harvard.edu/sherpa/).
 A [Quick Start Tutorial](http://nbviewer.ipython.org/github/sherpa/sherpa/tree/main/notebooks/SherpaQuickStart.ipynb)
 is included in the `notebooks` folder and can be opened with an `ipython notebook`.
 
+Acknowledging or Citing Sherpa
+==============================
+
+If you use Sherpa for work/research presented in a publication please cite the Sherpa papers:
+
+[Sherpa Paper 2024](https://arxiv.org/abs/2409.10400) ([ADS BibTex](https://ui.adsabs.harvard.edu/abs/2024arXiv240910400S/exportcitation))
+
+[Sherpa Paper 2007](https://articles.adsabs.harvard.edu/full/2007ASPC..376..543D) ([ADS BibTex](https://ui.adsabs.harvard.edu/abs/2007ASPC..376..543D/exportcitation) )
+
+[Sherpa Paper 2001](https://arxiv.org/abs/astro-ph/0108426) ([ADS BibTex](https://ui.adsabs.harvard.edu/abs/2001SPIE.4477...76F/exportcitation))
+
+If you are using AASTeX and plan to submit an article to one of the AAS journals, we recommend adding a \software{...} tag to your manuscript that cites Sherpa (see the [AASTeX guide](https://journals.aas.org/aastexguide/) for more information), e.g.:
+
+\software{Sherpa \citep{2001SPIE.4477...76F, 2007ASPC..376..543D, 2024arXiv240910400S}}
```

Comment:
Most LaTeX compilers don't like white space within a `\cite{}` command:
```suggestion
\software{Sherpa \citep{2001SPIE.4477...76F,2007ASPC..376..543D,2024arXiv240910400S}}
```

Also, I suggest to [squash the commits](https://stackoverflow.com/questions/5189560/how-do-i-squash-my-last-n-commits-together) since they don't really have separate information in them.